src/main/java/com/courtcanva/ccfranchise/dtos/StaffDto.java

```diff
@@ -0,0 +1,44 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class StaffDto {
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String lastName;
+
+    @Email
+    private String email;
+
+    @NotNull
+    private String phoneNumber;
+
+    @NotNull
+    private String address;
+
+    @NotNull
+    private Integer postcode;
+
+    @NotNull
+    private String state;
+
+    @NotNull
+    private String password;
```

Comment:
Format validation should be applied as well.

Reply:
now, I add extra validation for passwords (need one uppercase letter and number at least)